src/App.test.js

```diff
@@ -1,8 +1,13 @@
 import { render, screen } from '@testing-library/react';
+import {shallow} from 'enzyme';
 import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText('Team D App');
```

Comment:
Shouldn't this line be deleted from tests? We deleted the header **h1** that read 'Team D App' from _App.js_, so this test won't be passed.

Reply:
I think we agreed not to get involved in test files at this stage?

Reply:
Thx for advices, ill fix it tomorrow and pull it again ;)

Reply:
@WronskiN I don't think we mentioned tests but I agree that for now, we can skip them.

Reply:
Test files was my mistake, i forgot delete it